lib/KModal.vue

```diff
@@ -372,7 +372,10 @@
         }
         // focus has escaped the modal - put it back!
         if (!this.$refs.modal.contains(target)) {
-          this.focusModal();
+          this.$nextTick(() => {
+            // flush any pending DOM/focus updates
```

Comment:
May be nice to reference the issue https://github.com/learningequality/kolibri-design-system/issues/228, I don't think we will connect dots after a few months :)